tests/conftest.py

```diff
@@ -566,12 +566,12 @@ def relation_type(app):
 def relation_type_v(app, relation_type):
     """Relation type vocabulary record."""
     vocab = vocabulary_service.create(system_identity, {
-        "id": "cites",
+        "id": "iscitedby",
```

Comment:
Changing this to be able to test properly the mapping since `Cites` is only capitalized the first work... Wanted to check the proper camlecased.